cppcache/integration-test/testThinClientCqDurable.cpp

```diff
@@ -284,26 +283,25 @@ void RunFeederClient() {
 }
 
 void RunFeederClient1() {
-  CacheFactoryPtr cacheFactory = CacheFactory::createCacheFactory();
+  auto cacheFactory = CacheFactory::createCacheFactory();
   LOGINFO("Feeder connected to the Geode Distributed System");
 
-  CachePtr cachePtr = cacheFactory->create();
+  auto cachePtr = cacheFactory->create();
 
   LOGINFO("Created the Geode Cache");
 
-  RegionFactoryPtr regionFactory = cachePtr->createRegionFactory(PROXY);
+  auto regionFactory = cachePtr->createRegionFactory(PROXY);
 
   LOGINFO("Created the RegionFactory");
 
   // Create the Region Programmatically.
-  RegionPtr regionPtr = regionFactory->create("DistRegionAck");
+  auto regionPtr = regionFactory.create("DistRegionAck");
 
   LOGINFO("Created the Region Programmatically.");
 
   for (int i = 10; i < 20; i++) {
-    auto keyPtr =
-        std::dynamic_pointer_cast<CacheableKey>(CacheableInt32::create(i));
```

Comment:
What was this dynamic_pointer_cast doing?  How is it related to this review?

Reply:
Just casual cleanup of the code around required changes. In this case the cast was redundant.